packages/fxa-auth-server/test/local/metrics/amplitude.js

```diff
@@ -565,751 +565,83 @@ describe('metrics/amplitude', () => {
-        assert.equal(args[0].event_properties.email_type, 'login');
-      });
-    });
-
-    describe('email.verifyLoginCodeEmail.sent', () => {
-      beforeEach(() => {
-        return amplitude(
-          'email.verifyLoginCodeEmail.sent',
-          mocks.mockRequest({})
-        );
-      });
-
-      it('did not call log.error', () => {
-        assert.equal(log.error.callCount, 0);
-      });
-
-      it('called log.amplitudeEvent correctly', () => {
-        assert.equal(log.amplitudeEvent.callCount, 1);
-        const args = log.amplitudeEvent.args[0];
-        assert.equal(args[0].event_type, 'fxa_email - sent');
-        assert.equal(args[0].event_properties.email_type, 'login');
-      });
-    });
-
-    describe('email.verifyPrimaryEmail.bounced', () => {
-      beforeEach(() => {
-        return amplitude(
-          'email.verifyPrimaryEmail.bounced',
-          mocks.mockRequest({})
-        );
-      });
-
-      it('did not call log.error', () => {
-        assert.equal(log.error.callCount, 0);
-      });
-
-      it('called log.amplitudeEvent correctly', () => {
-        assert.equal(log.amplitudeEvent.callCount, 1);
-        const args = log.amplitudeEvent.args[0];
-        assert.equal(args[0].event_type, 'fxa_email - bounced');
-        assert.equal(args[0].event_properties.email_type, 'verify');
-      });
-    });
-
-    describe('email.verifyPrimaryEmail.sent', () => {
-      beforeEach(() => {
-        return amplitude(
-          'email.verifyPrimaryEmail.sent',
-          mocks.mockRequest({})
-        );
-      });
-
-      it('did not call log.error', () => {
-        assert.equal(log.error.callCount, 0);
-      });
-
-      it('called log.amplitudeEvent correctly', () => {
-        assert.equal(log.amplitudeEvent.callCount, 1);
-        const args = log.amplitudeEvent.args[0];
-        assert.equal(args[0].event_type, 'fxa_email - sent');
-        assert.equal(args[0].event_properties.email_type, 'verify');
-      });
-    });
-
-    describe('email.verifySecondaryEmail.bounced', () => {
-      beforeEach(() => {
-        return amplitude(
-          'email.verifySecondaryEmail.bounced',
-          mocks.mockRequest({})
-        );
-      });
-
-      it('did not call log.error', () => {
-        assert.equal(log.error.callCount, 0);
-      });
-
-      it('called log.amplitudeEvent correctly', () => {
-        assert.equal(log.amplitudeEvent.callCount, 1);
-        const args = log.amplitudeEvent.args[0];
-        assert.equal(args[0].event_type, 'fxa_email - bounced');
-        assert.equal(args[0].event_properties.email_type, 'secondary_email');
-      });
-    });
-
-    describe('email.verifySecondaryEmail.sent', () => {
-      beforeEach(() => {
-        return amplitude(
-          'email.verifySecondaryEmail.sent',
-          mocks.mockRequest({})
-        );
-      });
+    describe('email templates', () => {
+      const templates = require('../../../lib/senders/templates/_versions');
```

Comment:
Updated this test all check email templates vs email versions, which then tests the bounce/sent/delivered metrics. This gives us some assurance that we won't miss new templates for being reported to amplitude.